bundles/org.palladiosimulator.analyzer.slingshot.core/src/org/palladiosimulator/analyzer/slingshot/core/driver/SlingshotSimulationDriver.java

```diff
@@ -50,37 +50,32 @@ public SlingshotSimulationDriver(final SimulationEngine engine,
 
 	@Override
 	public void init(final SimuComConfig config, final IProgressMonitor monitor) {
-		if (this.initialized) {
-			return;
-		}
-		this.monitor = monitor;
-		this.config = config;
 
 		final List<Module> partitionIncludedStream = new ArrayList<>(behaviorContainers.size() + 1);
-
 		partitionIncludedStream.add(new SimulationDriverSubModule(monitor, config));
-
-		partitionIncludedStream.addAll(behaviorContainers);
+		//partitionIncludedStream.addAll(behaviorContainers); // TODO for some reason, this does not configure.
+		partitionIncludedStream.addAll(Slingshot.getInstance().getSimulationExtensions());
```

Comment:
ahh, i looked at the container's code (with a new focus), and now i understand how they do the distinquishing. also, i'm now less convinced about splitting the extension point. 
i'd put this up for discussion on friday. because both options have there pros and cons.

Reply:
also, i have an idea, why line 56 did not configure: because it only binds extensions previously added via `install` but does not redirect to the actual configure of the extension.. not perfectly sure though. as it's not realy important i will now stop thinking about it.